ipmi/Sensor_Discovery/scripts/ipmi-sensor-discovery.sh

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
+################################################################################
+# ipmi-sensor-discovery.sh - Zabbix IPMI sensor low-level discovery
+################################################################################
+#
+# Copyright (C) 2017 Radio Bern RaBe
+#                    Switzerland
+#                    http://www.rabe.ch
+#
+# This program is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public 
+# License as published  by the Free Software Foundation, version
+# 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License  along with this program.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+# Please submit enhancements, bugfixes or comments via:
+# https://github.com/radiorabe/rabe-zabbix
+#
+# Authors:
+#  Christian Affolter <c.affolter@purplehaze.ch>
+#
+# Description:
+# This script is intended to be used for Zabbix low-level discovery of IPMI
+# sensors, as Zabbix currently lacks LLD for IPMI item.
+#
+# It uses FreeIPMI's "ipmi-sensors" command to produce a list of available
+# sensors and transforms the output into the Zabbix LLD JSON format.
+# One is supposed to run this script as an external script on a Zabbix server
+# or proxy within a LLD rule.
+# The script lists all available sensors of a given IPMI host by default.
+# Optionally, the sensor list can be limited to one or multiple (comma separated)
+# senor types (see "ipmi-sensors --list-sensor-types").
+#
+# Refer to the awk command below for a list of LLD macros.
+#
+# Usage:
+# ipmi-sensor-discovery.sh IPMI-HOST IPMI-USER IPMI-PASSWORD [IPMI-SENSOR-TYPES]
+#
+set -o pipefail
+
+export LC_ALL="C"
+
+IPMI_SENSORS_CMD="/usr/sbin/ipmi-sensors"
+AWK_CMD="/usr/bin/awk"
```

Comment:
How about using `AWK_CMD=${AWK-/usr/bin/awk}` (and something similar for `ipmi-sensors`) so these declarations make a real difference compared to just hardcoding the paths?